### PR TITLE
Update helm publish script and release md file per split repo

### PR DESCRIPTION
**Pull request by kumargauravsharma**
Issue #, if available:

https://github.com/aws-controllers-k8s/community/blob/main/scripts/helm-publish-charts.sh script needs to be updated to publish single service controller and need to adjust its arguments and paths per split repositories.

Description of changes:
* Renamed `helm-publish-charts.sh`  to `helm-publish-chart.sh`
* Updated `helm-publish-chart.sh` to publish single service controller and need to adjust its arguments and paths per split repositories.
* Updated release.md file for helm publish step.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/contents/dev-docs/release.md [hunk 1]
```diff
@@ -146,10 +146,15 @@ Login succeeded
     something similar.
 
 Now publish all the controller images to the ECR public repository for
-controller images using the `scripts/helm-publish-charts.sh` script:
+controller image using the `scripts/helm-publish-charts.sh` script for each service:
+
+```bash
+for SERVICE in apigatewayv2 sns;\
```
Comment (jaypipes) [inline docs/contents/dev-docs/release.md:152]: typically we place the `do` after the `;` and leave off the `\`, like so:

```bash
for SERVICE in apigatewayv2 sns; do
```

**Review by jaypipes — CHANGES_REQUESTED** [review]
Nice work on this @kumargauravsharma. Just a couple tiny nits to fix up and this is good to go.

File: docs/contents/dev-docs/release.md [hunk 2]
```diff
@@ -146,10 +146,15 @@ Login succeeded
     something similar.
 
 Now publish all the controller images to the ECR public repository for
-controller images using the `scripts/helm-publish-charts.sh` script:
+controller image using the `scripts/helm-publish-charts.sh` script for each service:
```
[the remark this answers is not in the document]
Comment (kumargauravsharma) [reply docs/contents/dev-docs/release.md]: Updated the document.

File: docs/contents/dev-docs/release.md [hunk 3]
```diff
@@ -146,10 +146,15 @@ Login succeeded
...
+
+```bash
+for SERVICE in apigatewayv2 sns;\
+    do RELEASE_VERSION=v0.0.1 ./scripts/helm-publish-charts.sh $SERVICE;
```
[the remark this answers is not in the document]
Comment (kumargauravsharma) [reply docs/contents/dev-docs/release.md]: Updated the document.

File: docs/contents/dev-docs/release.md (hunk 1 above)
Comment (kumargauravsharma) [reply docs/contents/dev-docs/release.md]: Updated the document.

**Review by jaypipes — APPROVED** [review]
Great work, thank you @kumargauravsharma!

**Pull request merged**